examples/plotting/tap.py

```diff
@@ -1,3 +1,14 @@
+''' An interactive numerical tap plot based on a simple Python array of data.
+    You can select any datapoint by tapping on it. 
+    This highlights that datapoint and displays all other datapoints in a faded color.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure, bokeh.plotting.show, bokeh.models.TapTool
```

Comment:
```suggestion
    :apis: bokeh.plotting.figure, bokeh.io.show, bokeh.models.TapTool
```
And this one needs to be for https://docs.bokeh.org/en/latest/docs/reference/io.html#bokeh.io.show - sorry for the multiple requests!

Reply:
The best way to verify these api paths is looking up the method or class in the reference guide https://docs.bokeh.org/en/latest/docs/reference.html